### PR TITLE
Betting UI - Big Tabs and Tab Labels

### PR DESCRIPTION
**Pull request by rubyweb3**
https://github.com/AugurProject/augur/issues/6150
https://github.com/AugurProject/augur/issues/6151

Example:
It's an example
Normal
Screen Shot 2020-03-03 at 6 38 30 PM

Screen Shot 2020-03-03 at 6 38 46 PM

Large
Screen Shot 2020-03-03 at 7 00 32 PM
Screen Shot 2020-03-03 at 7 00 25 PM
Screen Shot 2020-03-03 at 7 00 04 PM

Small
Screen Shot 2020-03-03 at 7 03 55 PM
Screen Shot 2020-03-03 at 7 03 42 PM
Screen Shot 2020-03-03 at 7 03 17 PM

### CODE REVIEW

File: packages/augur-ui/src/modules/common/selection.tsx [hunk 1]
```diff
@@ -390,6 +401,50 @@ export const PillSelection = ({
   );
 }
 
+export const Tab = ({
```
Comment (rubyweb3) [inline packages/augur-ui/src/modules/common/selection.tsx:405]: @jfarid27 Is this PR moving in the right direction? Active and disable state are still in progress. I will add it soon

Comment (jfarid27) [reply packages/augur-ui/src/modules/common/selection.tsx:405]: @rubyweb3 I'm not sure. I actually haven't looked at this in a while. Which issue are you referring to?

Comment (rubyweb3) [reply packages/augur-ui/src/modules/common/selection.tsx:405]: Sorry. @ wrong person

File: packages/augur-ui/src/modules/common/selection.styles.less [hunk 2]
```diff
@@ -450,6 +450,177 @@
...
+      }
+    }
+  }
+
+  &.TabSmall {
+    height: @size-26;
+
+    > li {
+      > button {
+        .mono-12-bold;
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/common/selection.styles.less]: I think this one is actually semi-bold

**Review by JohnDanz — CHANGES_REQUESTED** [review]
one minor change

File: packages/augur-ui/src/modules/common/selection.styles.less (hunk 2 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/common/selection.styles.less]: Updated. Thank you.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**